src/Fhir.Anonymizer.Tool/Program.cs

```diff
@@ -17,6 +18,8 @@ class Options
             public string OutputFolder { get; set; }
             [Option('c', "configFile", Required = false, Default = "configuration-sample.json", HelpText = "Anonymizer configuration file path.")]
             public string ConfigurationFilePath { get; set; }
+            [Option('m', "mappingFile", Required = false, Default = "mapping-ids.dat", HelpText = "Resource Id mapping file path.")]
+            public string MappingFilePath { get; set; }
```

Comment:
This parameter is used for output? It seems we never load it.

Reply:
Besides, it seems transforming id is always done by engine. It is not optional right? And do we need the same logic for adf?

Reply:
Yes. This file option is for saving mapping for re-Id purpose. Should be add in adf too

Reply:
Got it. If it is used for output, can we add some explanation like "Resource id mapping file saved from id transformation"?